frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -30,15 +32,47 @@ interface IAdvancedConfigFormData {
 }
 
 interface IAdvancedConfigFormErrors {
-  host_expiry_window?: string | null;
+  mdmAppleServerURL?: string | null;
+  domain?: string | null;
+  hostExpiryWindow?: string | null;
 }
 
+const validateFormData = ({
+  mdmAppleServerURL,
+  domain,
+  hostExpiryWindow,
+  enableHostExpiry,
+}: IAdvancedConfigFormData) => {
```

Comment:
is this typing right? arent we returning the form errors and not the data with this function?

Reply:
Ah, yea it's right. Thanks for checking.